lib/ValueAnonymizers/Credentials.php

```diff
@@ -24,7 +24,15 @@ public function anonymize(Value $value, array $row): AnonymizedValue
             return AnonymizedValue::fromOriginalValue($value);
         }
 
-        $anonymizedEscapedValue = $this->stringHash->hashKeepFormat($value->getUnEscapedValue(), true);
+        $string = $value->getUnEscapedValue();
+
+        if (strlen($string) <= 10) {
```

Comment:
if length is 10 or even 9, wouldn't taking the first 10 chars of the hexa hash actually reduce the variation?
I think we should use `< 10` or maybe even `< 9` instead of `<= 10`

Reply:
Done